code/game/objects/items/storage/firstaid.dm

```diff
@@ -98,6 +98,10 @@
 /obj/item/storage/firstaid/regular/empty/fill_preset_inventory()
 	return
 
+/obj/item/storage/firstaid/regular/response
+	required_skill_for_nest_opening = SKILL_MEDICAL
+	required_skill_level_for_nest_opening = SKILL_MEDICAL_DEFAULT
```

Comment:
Change the medkit desc to not have the skill bit

Reply:
Done @Zonespace27 :D